packages/node-sdk/src/client.ts

```diff
@@ -86,7 +86,7 @@ interface ContextWithTracking extends Context {
    * Enable tracking for the context.
    * If set to `false`, tracking will be disabled for the context. Default is `true`.
    */
-  enableTracking: boolean;
+  enableTracking?: boolean;
```

Comment:
This is technically only needed if users will prepare a context object and pass it around to functions. If they'd call the functions directly with the object notation the interface takes care of the default. But it's better to have it undefined as well for those cases